src/plugins/mer/meremulatoroptionswidget.cpp

```diff
@@ -376,6 +376,11 @@ void MerEmulatorOptionsWidget::onAddButtonClicked()
     };
     Sdk::createEmulator(dialog.selectedVmUri(), &loop, whenDone);
     loop.exec();
+    if (!emulator) {
+        QMessageBox::warning(this, tr("Could not add emulator"),
+                             tr("The selected virtual machine is not suitable to be used as an emulator"));
```

Comment:
```suggestion
        QMessageBox::warning(this,
                tr("Could not add a %1 Emulator").arg(Sdk::osVariant()),
                tr("The selected virtual machine is not suitable to be used as a %1 Emulator")
                    .arg(Sdk::osVariant()));
```

I would also consider saying something like "could not be configured as" instead of "is not suitable to be used as" - as a less specific judgement (maybe the VM is an emulator VM and just some small bits are missing).